dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -7,8 +7,13 @@ import {
 	brandText,
 	from,
 	headline,
+	palette,
```

Comment:
We never import this property from Source, (I think it's actually a mistake that it's available to be imported). You can just import `neutral` directly (which means you import a lot less code)